src/main/java/net/neoforged/neoforge/common/data/internal/NeoForgeDamageTypeTagsProvider.java

```diff
@@ -92,6 +118,22 @@ protected void addTags(HolderLookup.Provider lookupProvider) {
         tagWithOptionalLegacy(Tags.DamageTypes.NO_FLINCH);
     }
 
+    private TagAppender<DamageType> addAsVanilla(ResourceKey<DamageType> entry) {
```

Comment:
Just for quick reference in the future, add a comment (even if not a javadoc one) explaining what this method does -- returns an appender which appends to the same tags that (directly) contain the given entry.